pkg/apis/serving/v1/revision_defaults_test.go

```diff
@@ -94,8 +96,45 @@ func TestRevisionDefaulting(t *testing.T) {
 			},
 		},
 	}, {
-		name: "with service links `true`",
+		name: "with context, in create",
```

Comment:
the name of this test doesn't make it clear what we're testing/expecting

Reply:
Done.